[SP-10275] update readme (#62)
Losant/losant-cli: README.md

### Login

Before you run any other commands, you must run `losant login` to authenticate with your Losant account. This command checks to see if your account is linked to a Single Sign-On (SSO) provider. If so, the command will prompt for a User Token; otherwise it will prompt for the password (and optionally your two-factor code) for your Losant account. After either is given successfully, the command will store

> the authentication token on your computer. On this command you can optionally set the LOSANT_API_URL as an enviroment variable e.g. `LOSANT_API_URL=<api.private.install> losant login`, by default the CLI will use `https://api.losant.com` as the API URL. This will allow you to login to multiple users across Losant installs. If you are logged into multiple Losant installs when you configure a directory, you will be asked which API token to use to access the application. And from then on any request for that application will use the same API URL.

Reviewer: ```suggestion
the authentication token on your computer. With this command, you can optionally set `LOSANT_API_URL` as an environment variable; e.g. `LOSANT_API_URL=<api.private.install> losant login`. By default the CLI will use `https://api.losant.com` as the API URL. This will allow you to log in across Losant installations. If you are logged in to multiple Losant installations when you configure a directory, you will be asked which API token to use to access the application. From then on, any request for that application will use the same API URL.
```

### Set-token

> The set-token command, `losant set-token`, is an alternative way to log in for those users whose account is linked to a Single Sign-On (SSO) provider. This will take the given token, verify that token against the API, and then set it on your user configuration file.  On this command you can optionally set the LOSANT_API_URL as an enviroment variable e.g. `LOSANT_API_URL=<api.private.install> losant set-token`, by default the CLI will use `https://api.losant.com` as the API URL. This will allow you to set a token for any Losant private installation, and allow you to set up multiple tokens (or use the login command) for different Losant installs. If you do login to multiple Losant API instances, when configuring a directory for an application, the configure command will ask which API to use to find the application. And from then on any request for that application will use the same API URL.

Reviewer: ```suggestion
The set-token command, `losant set-token`, is an alternative way to log in for those users whose account is linked to a Single Sign-On (SSO) provider. This will take the given token, verify that token against the API, and then set it on your user configuration file.  With this command, you can optionally set the `LOSANT_API_URL` as an environment variable; e.g. `LOSANT_API_URL=<api.private.install> losant set-token`. By default the CLI will use `https://api.losant.com` as the API URL. This will allow you to set a token for any Losant private installation, and will allow you to set up multiple tokens (or use the `login` command) for different Losant installations. If you do log in to multiple Losant installations, when configuring a directory for an application, the `configure` command will ask which API to use to find the application. From then on, any request for that application will use the same API URL.
```
